### PR TITLE
Documented design decisions

### PR DESCRIPTION
**Pull request by joeribekker**

### CODE REVIEW

**Comment by shea-maykinmedia** [comment]
@joeribekker 

I've added the mappings for the models and values.  

When I could not find the value from Stuf-BG I would return either "string", 0, or True and so this is what I put in the StUF-BG attribute column though it's not really a StUF-BG attribute.  I could put these values in the Design Decision column but I thought this might be unclear. 

For some values that are being converted I wrapped the value in either `boolean` or `integer` to be clear that the conversion was taking place and what sort of conversion.

I didn't put too much in the Design Decision Remarks column since most stuff is just a direct mapping or is returning a 'default' value. 

Let me know if there is anything you want to be added or changed.

File: docs/backends/stufbg/implementation.rst [hunk 1]
```diff
@@ -164,49 +163,50 @@ API attribute                                                           StUF-BG
...
 `verblijfplaats.datumInschrijvingInGemeente`                            `True`                                                                  TODO
 `verblijfplaats.datumVestigingInNederland`                              `True`                                                                  TODO
+
 `verblijfplaats.functieAdres`                                           `True`                                                                  TODO
 `verblijfplaats.gemeenteVanInschrijving`                                `True`                                                                  TODO
-`verblijfplaats.inOnderzoek.huisletter`                                 `True`                                                                  TODO
-`verblijfplaats.inOnderzoek.huisnummer`                                 `True`                                                                  TODO
-`verblijfplaats.inOnderzoek.huisnummertoevoeging`                       `True`                                                                  TODO
+`verblijfplaats.inOnderzoek.huisletter`                                 *(`inOnderzoek.groepsnaam` == `Postadres`)*
```
Comment (joeribekker) [inline docs/backends/stufbg/implementation.rst:169]: This should reflect `Verblijfsplaats`, like all elements with that group.

File: docs/backends/stufbg/implementation.rst [hunk 2]
```diff
@@ -164,49 +163,50 @@ API attribute                                                           StUF-BG
...
+`verblijfplaats.inOnderzoek.huisnummer`                                 *(`inOnderzoek.groepsnaam` == `Postadres`)*
+`verblijfplaats.inOnderzoek.huisnummertoevoeging`                       *(`inOnderzoek.groepsnaam` == `Postadres`)*
 `verblijfplaats.inOnderzoek.identificatiecodeNummeraanduiding`          `True`                                                                  TODO
 `verblijfplaats.inOnderzoek.identificatiecodeAdresseerbaarObject`       `True`                                                                  TODO
 `verblijfplaats.inOnderzoek.landVanwaarIngeschreven`                    `True`                                                                  TODO
 `verblijfplaats.inOnderzoek.locatiebeschrijving`                        `True`                                                                  TODO
 `verblijfplaats.inOnderzoek.naamOpenbareRuimte`                         `True`                                                                  TODO
```
Comment (joeribekker) [inline docs/backends/stufbg/implementation.rst]: So, all of these should also be taken from `inOnderzoek.groepsnaam == `Verblijftplaats`

File: docs/backends/stufbg/implementation.rst [hunk 3]
```diff
@@ -164,49 +163,50 @@ API attribute                                                           StUF-BG
...
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.jaar`                  1900                                                                    Fixed value since not in StUF-BG and cannot be `null`.
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.maand`                 1                                                                       Fixed value since not in StUF-BG and cannot be `null`.
 `gezagsverhouding.indicatieCurateleRegister`                            `True`                                                                  TODO
-`gezagsverhouding.indicatieGezagMinderjarige`                           `ouder1`                                                                TODO
+`gezagsverhouding.indicatieGezagMinderjarige`                           `string`                                                                TODO
 `gezagsverhouding.inOnderzoek.indicatieCurateleRegister`                `True`                                                                  TODO
 `gezagsverhouding.inOnderzoek.indicatieGezagMinderjarige`               `True`                                                                  TODO
```
Comment (joeribekker) [inline docs/backends/stufbg/implementation.rst]: These reflect `inOnderzoek.groepsnaam` == `Gezagsverhouding`

File: docs/backends/stufbg/implementation.rst [hunk 4]
```diff
@@ -164,49 +163,50 @@ API attribute                                                           StUF-BG
...
+`verblijfplaats.inOnderzoek.woonplaatsnaam`                             *(`inOnderzoek.groepsnaam` == `Postadres`)*
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.dag`                   1                                                                       Fixed value since not in StUF-BG and cannot be `null`.
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.datum`                 "01-01-1900"                                                            Fixed value since not in StUF-BG and cannot be `null`.
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.jaar`                  1900                                                                    Fixed value since not in StUF-BG and cannot be `null`.
+`verblijfplaats.inOnderzoek.datumIngangOnderzoek.maand`                 1                                                                       Fixed value since not in StUF-BG and cannot be `null`.
 `gezagsverhouding.indicatieCurateleRegister`                            `True`                                                                  TODO
```
Comment (joeribekker) [inline docs/backends/stufbg/implementation.rst:185]: This can be `indicatieCurateleRegister`

File: docs/backends/stufbg/implementation.rst [hunk 5]
```diff
@@ -164,49 +163,50 @@ API attribute                                                           StUF-BG
...
+`gezagsverhouding.inOnderzoek.datumIngangOnderzoek.dag`                 1                                                                       Fixed value since not in StUF-BG and cannot be `null`.
+`gezagsverhouding.inOnderzoek.datumIngangOnderzoek.datum`               "01-01-1900"                                                            Fixed value since not in StUF-BG and cannot be `null`.
+`gezagsverhouding.inOnderzoek.datumIngangOnderzoek.jaar`                1900                                                                    Fixed value since not in StUF-BG and cannot be `null`.
+`gezagsverhouding.inOnderzoek.datumIngangOnderzoek.maand`               1                                                                       Fixed value since not in StUF-BG and cannot be `null`.
 `verblijfstitel.aanduiding.code`                                        `vbt.aanduidingVerblijfstitel`
 `verblijfstitel.aanduiding.omschrijving`                                `string`                                                                TODO
```
Comment (joeribekker) [inline docs/backends/stufbg/implementation.rst:194]: This should be obtained from a static mapping table, that can be found here (code-description): https://publicaties.rvig.nl/dsresource?objectid=4801&type=org

File: docs/backends/stufbg/implementation.rst [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+`nationaliteit.nationaliteit.code`                                      `inp.heeftAlsNationaliteit.gerelateerde.code`
+`nationaliteit.nationaliteit.omschrijving`                              `inp.heeftAlsNationaliteit.gerelateerde.omschrijving`
+`nationaliteit.redenOpname.code`                                        REDEN_CODE(`inp.heeftAlsNationaliteit.inp.redenVerkrijging`)                       TODO
+`nationaliteit.redenOpname.omschrijving`                                `inp.heeftAlsNationaliteit.inp.redenVerkrijging`
+`nationaliteit.inOnderzoek.aanduidingBijzonderNederlanderschap`         *(`inOnderzoek.elementnaam` == `aanduidingBijzonderNederlanderschap`)*             One of the multiple inOnderzoek occurrances matches
+`nationaliteit.inOnderzoek.nationaliteit`                               *(`inOnderzoek.groepsnaam` == `Nationaliteit`)*                                    One of the multiple inOnderzoek occurrances matches
+`nationaliteit.inOnderzoek.redenOpname`                                 `True`                                                                             TODO
```
[the remark this answers is not in the document]
Comment (shea-maykinmedia) [reply docs/backends/stufbg/implementation.rst:94]: Yes, the redenOpname within inOnderzoek is a boolean value.

**Review by shea-maykinmedia — APPROVED** [review]

**Comment by shea-maykinmedia** [comment]
@joeribekker Since you opened the pull request there isn't a way for me to ask for a review ;) but I think this is good to merge.  Can you double check it when you have time.

**Pull request merged**